### PR TITLE
Fix: issue-45 diamond to square

### LINKED ISSUE #45: diamond to square
I am trying to use the diamond shape but it's converted into a square. Actually, I am using className prop on the component for adding some cool animations.
It stays for some time as a diamond and then it jumps back to square as shown in the image below!
Screenshot 2022-03-02 at 7 58 53 PM

Screenshot 2022-03-02 at 7 58 26 PM
Screenshot 2022-03-02 at 7 58 21 PM

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
I work on fixing issue #45 which converts diamond shape in to square shape on animation.
Closes #45 

Working example:- Please look right top corner for diamond shape

https://user-images.githubusercontent.com/71748675/158029836-3b14c96e-202a-4159-9eba-70326f7e4a66.mov

@Sachin-chaurasiya @Spyware007 @ashutosh1919

### CODE REVIEW

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
Please apply `upKeyframe` in `shapes.tsx` for diamond. It will help see whether the problem is fixed or not.

**Comment by ShaileshParmar11** [comment]
> Please apply `upKeyframe` in `shapes.tsx` for diamond. It will help see whether the problem is fixed or not.

Hi @ashutosh1919, I have added `upKeyFrame` kindly check and let me know if anything more need to address for this issue.

File: src/components/shapes.tsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ export const Shapes = () => {
       <Diamond
         color="linear-gradient(225deg, #fed7aa, #fb923c)"
         right={[`40px`, `50px`, `60px`, `120px`]}
-        size="30px"
+        size="50px"
```
Comment (ashutosh1919) [inline src/components/shapes.tsx:46]: Why increase in size? please keep the size same.

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]

**Comment by ashutosh1919** [comment]
@all-contributors please add @ShaileshParmar11 for code

**Pull request merged**